docs/2_2_common_schema.adoc

```diff
@@ -1402,6 +1402,13 @@ It is not allowed to combine <<outputClock,`output clocks`>> with periodic or st
 
 The attribute <<clockReference>> must not be used if <<causality>> = <<output>> for a variable.
 
+|`clockElementIndex`
+|
+[[clockElementIndex,`clockElementIndex`]]
+The optional attribute <<clockElementIndex>> is used in conjuction with the <<clockReference>> attribute to define the element index of the <<clock,`clocks`>> this variable is assigned to for array clock variables.
+The element index is the 1-based index of the element (with serialization of indeces for multi-dimensional arrays in the usual row-major order).
```

Comment:
Shouldn't the indexing be consistent with the rest of the API (i.e. C-style)?

Reply:
This is consistent with `elementIndicesIndependents` - but I agree this is odd, non-C-style and will likely lead to errors. Using 0 as the catch-all is arbitrary. Since size_t is unsigned, -1 is out. MAX_SIZE_T does not exist (or does it?). But can we define it in fmi3PlatformTypes.h?
Better ideas, anyone?

Reply:
@t-sommer, @andreas-junghanns Not that I like this convention, and for the clocks we might not actually need this, but if we do this thing in the dependencies API, we'd better stay consistent to it. If we want to change this, I think we should do this in a separate pull request, where we change both sites.

Don't think that I have any better ideas though, since the -1 (UNSIGNED MAX) thingy seems problematic, since it does not work very well for the XML part, really.